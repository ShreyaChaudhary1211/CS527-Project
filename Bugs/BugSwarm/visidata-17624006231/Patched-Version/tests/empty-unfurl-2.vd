sheet	col	row	longname	input	keystrokes	comment
global		unfurl_empty	set-option	True		
			open-file	sample_data/test.jsonl	o	
test	key2		addcol-expr	list(key2.strip())	=	create new column from Python expression, with column names as variables
test	list(key2.strip())		unfurl-col			row-wise expand current column of lists (e.g. [2]) or dicts (e.g. {3}) within that column
