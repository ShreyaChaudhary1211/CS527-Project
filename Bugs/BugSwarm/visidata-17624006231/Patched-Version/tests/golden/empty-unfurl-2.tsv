key1	key2	list(key2.strip())_key	list(key2.strip())_value	qty	amt	
2016-01-01 11:00:00	foo	0	f	1		
2016-01-01 11:00:00	foo	1	o	1		
2016-01-01 11:00:00	foo	2	o	1		
2016-01-01 1:00		#ERR	#ERR	2	3	
	baz	0	b	4	43.2	
	baz	1	a	4	43.2	
	baz	2	z	4	43.2	
#ERR	#ERR	#ERR	#ERR	#ERR	#ERR	#ERR
2017-12-25 8:44	bar	0	b	16	.3	
2017-12-25 8:44	bar	1	a	16	.3	
2017-12-25 8:44	bar	2	r	16	.3	
	baz	0	b	32	3.3	
	baz	1	a	32	3.3	
	baz	2	z	32	3.3	
2018-07-27 4:44		#ERR	#ERR	64	9.1	
2018-07-27 16:44	bar	0	b			
2018-07-27 16:44	bar	1	a			
2018-07-27 16:44	bar	2	r			
2018-07-27 18:44	baz	0	b	256	.01	
2018-07-27 18:44	baz	1	a	256	.01	
2018-07-27 18:44	baz	2	z	256	.01	
2018-10-20 18:44	foo	0	f	30.0	.01	
2018-10-20 18:44	foo	1	o	30.0	.01	
2018-10-20 18:44	foo	2	o	30.0	.01	
